build.gradle.kts

```diff
@@ -627,7 +625,7 @@ tasks {
   }
 
   val compileTrinoConnector by registering {
-    dependsOn("trino-connector:copyLibs")
+    dependsOn("trino-connector:trino-connector:copyLibs")
```

Comment:
Can you please check it the LICENSE, NOTICE, and DISCLAIMER are correctly copied to the release package for trino-connector, web and gravitino package? Since the package location is changed, I guess it will be failed to locate the file, can you please check and fix?

Reply:
Also check the `compileDistribution` and `assembleDistribution`, to see if they still work or not?

Reply:
I have checked them all, they works well